submissions/hello_world/README.md

```diff
@@ -0,0 +1,22 @@
+# Hello World
+
+This is an example hello world program.
+It is s simple executable python3 script that will print `Hello, World!` when run.
+
+## Usage
+
+To use this program you must have `python3` installed.
+
+Once python is installed the following commands can be used to run it.
+
+```bash
+$ cd submissions/hello_world
+```
+
+```bash
+$ ./HelloWorld.py
+```
+
+## Why I Added it
+
+This program is being added as an example contribution.
```

Comment:
You should add a list of contributors in this README

Reply:
fixed in ceeb583